### ISSUE TITLE
[iOS] Text input doesn't open keyboard

### ISSUE THREAD

**Issue by nighto** [hrydgard/ppsspp#18449]
### Game or games this happens in

None

### What area of the game / PPSSPP

On Settings, anytime there is a text input, you touch the input field and the virtual keyboard does not pop-up. For instance, the username / password for RetroAchievements, or the "change PSP nickname" option.

![image](https://github.com/hrydgard/ppsspp/assets/397851/b7bb5b5a-559b-4bbb-befa-f8af110778c3)

### What should happen

The virtual keyboard should popup, either the native or a custom-baked like RetroArch does.

### Logs

(Not sure how to grab logs from iOS, I can update later.)

### Platform

iOS

### Mobile phone model or graphics card

iPhone 11

### PPSSPP version affected

v1.16.6-356-g4d94b7006 (from W.MS' latest ipa)

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by nighto** [comment]
Not sure if really helpful but if a bluetooth joypad is connected, it is possible to move the text cursor with the text buttons. However it is not possible to enter text. Maybe with a bluetooth keyboard?

**Comment by nighto** [comment]
Just tested with a bluetooth keyboard connected to the iPhone and it is really not possible to enter text.

**Comment by nighto** [comment]
Googling around seems it is not a iOS-only specific bug, folks on Steam Deck have related the same issue: https://www.reddit.com/r/RetroAchievements/comments/16lil4u/ppsspp_standalone_retroachievement_login_issue_on/

**Comment by nighto** [comment]
Seems to be a duplicate of #18222, not sure if I should close this one as opening a virtual keyboard on iOS should be simpler than on Steam Deck (or not, don't know).

**Comment by nighto** [comment]
Found a workaround to use RetroAchievements for now: use PPSSPP on computer once to login on RetroAchievements, quit PPSSPP on computer, then copy `ppsspp/memstick/PSP/SYSTEM/ppsspp.ini` as well as the `ppsspp_retroachievements.dat` file to iPhone (using iTunes on Windows for instance). Then, you need to open Files app, open PPSSPP folder, three dots on top right corner, Visualization options, Show all extensions, then open `PSP/SYSTEM` folder, rename `ppsspp.ini` to `ppsspp.ini.txt`, open the `ppsspp.ini` you copied, copy the whole `[Achievements]` block to your `ppsspp.ini.txt` file, rename it back to `ppsspp.ini`, then move `ppsspp_retroachievements.dat` to this `PSP/SYSTEM` folder. After all of this you can open PPSSPP on your iPhone and RetroAchievements will be already logged in.

**Comment by hrydgard** [comment]
Yes, this is a duplicate. Closing, I'll add a little note in #18222 too.

**Comment by Fapp3r3Lit3** [comment]
You can set it up with "Accessability- Voice" 

I have it programmed if
A certain word is said, it can, unlock my device. Delete every message/notes/ and factory restore it self.
